src/webgpu/api/validation/resource_usages/textureUsageInRender.spec.ts

```diff
@@ -232,3 +237,133 @@ g.test('shader_stages_and_visibility')
       encoder.finish();
     });
   });
+
+g.test('replaced_binding')
```

Comment:
Can we have these tests added to the test plan above?

Reply:
Done